timescaledb/how-to-guides/connecting/azure-data-studio.md

```diff
@@ -101,30 +81,35 @@ After successfully connecting, your server opens in the `SERVERS`sidebar.
 1.  Start `Azure Data Studio`.
 1.  Click the `New Connection` icon in the `SERVERS`page.
 1.  In the form that pops up, go to `Connection type` and select `PostgreSQL`
-   from the drop-down. To connect to the TimescaleDB hosted on your local
-   machine or on a remote server. Fill in the fields using the server name, user
-   name, and password for your Managed Service for TimescaleDB service.
+    from the drop-down. To connect to the TimescaleDB hosted on your local
+    machine or on a remote server. Fill in the fields using the server name, user
+    name, port, and password for your Managed Service for TimescaleDB service. For
+    description of the settings in `Connection Details`,
+    see [Connection Detais][connection-details].
 
    <img class="main-content__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/mst_connection_screen.png"alt="Connection screen for Managed Service for TimescaleDB service"/>
 
-   | Setting       | Example value | Description |
-   | ------------ | ------------------ | ------------------------------------------------- |
-   |Server name|<REMOTE_HOST>.timescaledb.io|Host name of your Managed Service for TimescaleDB service|
-   |Authentication type|Password|The authentication type to log in with|
-   |User name|tsdbadmin|The user name to log into your Managed Service for TimescaleDB service|
-   |Password| *password* |The password to log into your Managed Service for TimescaleDB service|
-   |Remember password| *Check* |Check this box if you don't want to enter the password each time you connect|
-   |Database name|defaultdb|This option connects to the `defatltdb` database|
-   |Server group| \<Default\> |This option lets you assign this connection to a specific server group you create|
-   |Name (optional)| *leave blank* |This option lets you specify a friendly name for your server|
-
 1.  Select `Connect`.
 
 After successfully connecting, your server opens in the `SERVERS`sidebar.
 
 </procedure>
 
+### Settings in the `Connection Details`
+
+   |Setting|Example value|Description|
+   | ------------ | ------------------ | ------------------------------------------------- |
+   |Server name|localhost, postgresql.example.com, <REMOTE_HOST>.tsdb.cloud.timescale.com, <REMOTE_HOST>.timescaledb.io |The fully qualified server name. The host name of your Timescale Cloud service or Managed Service for TimescaleDB service.|
+   |Authentication type|Password|The authentication type to log in with|
+   |User name|postgres, tsdbadmin|The user name you want to log in with|
+   |Password|*password*|The password for the account you are logging in with|
+   |Remember password|*Check*|Check this box if you don't want to enter the password each time you connect.|
+   |Database name|\<Default\>, defaultdb|This option connnects the database that you specify.The default database on your Managed Service for TimescaleDB service is `defaultdb`.|
```

Comment:
Is it still `defaultdb`? I really have no idea, but I seem to recall Mike saying it's `tsdb`

Reply:
I just picked the name that appeared in the service. I have no idea. @alexthinath  could you please confirm the name of the default database in mst.

Reply:
@alexthinath confirmed that it was `tsdb` . Just updated it to `tsdb`.

Reply:
updated it to `tsdb`